frontend/src/modules/Catalog/components/GlossarySearch.js

```diff
@@ -319,4 +319,3 @@ GlossarySearch.propTypes = {
   setQuery: PropTypes.func.isRequired,
   matches: PropTypes.array.isRequired
 };
-export default GlossarySearch;
```

Comment:
For 2.0.1: we should rename `GlossarySearch` and `GlossarySearchComponent` to be more consistent, i.e., when we read the name, we should have an idea of what each entity is: what is `GlossarySearch` vs `GlossarySearchComponent`? Why does one have a qualifier and the other doesn't?

Reply:
renamed to `GlossarySearchWrapper` and `GlossarySearchUI`